key-service.js

```diff
@@ -23,11 +24,13 @@ function factory(
     options = options || {};
     var identityId;
     var url;
+    var params = $httpParamSerializer(options.params)
     if(options.identity) {
-      url = basePath + '?owner=' + encodeURIComponent(options.identity.id);
+      url = basePath +
+        '?owner=' + encodeURIComponent(options.identity.id) + '&' + params;
       identityId = options.identity.id;
     } else {
-      url = basePath;
+      url = basePath + '?' + params;
       identityId = null;
```

Comment:
Probably better to use some URL formatting lib/API here vs. string concatenation, but it will do for now.